app/controllers/registered_applications_controller.rb

```diff
@@ -0,0 +1,60 @@
+class RegisteredApplicationsController < ApplicationController
+  def index
+    @registered_applications = RegisteredApplication.all
```

Comment:
Do you think it makes sense for a user to be able to see just his registered applications?

`current_user.registered_applications`